### ISSUE TITLE
Request for the Release of the SFT Dataset for Distillation

### ISSUE THREAD

**Issue by jakeju92** [deepseek-ai/DeepSeek-R1#14]
Hi,

Congratulations on such a fantastic piece of work!

In Section 2.4 of your paper, you mention employing an **800k curated dataset** for SFT on the LLama and Qwen models. I was wondering if there is any possibility of open-sourcing this dataset? It would be immensely valuable for the community and could greatly aid related research efforts.

**Comment by corbt** [comment]
Also, could you clarify what "distillation" means in this context? Were the distilled models simply trained using standard SFT on the generated traces/answers from Deepseek R1? Or were they trained with logit-level distillation of some kind?

**Comment by hancheolcho** [comment]
@corbt As I understand, they ran SFT on the 800k sft data.

![Image](https://github.com/user-attachments/assets/dd834861-0ab1-4696-a2a9-80a07d376526)

**Comment by EoinKenny** [comment]
Yes please this is exactly what I need too, i want to train interpretable by design LLMs that can reason with good performance, this would be perfect to help that research goal. Please open source this data so we can replicate the distillation process.

**Comment by AnnaGolubeva** [comment]
+1

**Comment by lw309637554** [comment]
+1

**Comment by DA21S321D** [comment]
mysterous so-called "distillation" i will never know, after searching the entire web tutorial :(
1.does the distillation **compute the loss between teacher and student logits** ?
2.or does it just collect a **high-quality SFT data** using rejection sampling and then **perform SFT** on the small parameters model?
3.does distillation related to lora? ( i have seen many tutorials teaching us reproducing "distillation" by **lora fine-tuning**)
